antsibull/utils/get_pkg_data.py

```diff
@@ -0,0 +1,14 @@
+# Author: Felix Fontein <felix@fontein.de>
+# Author: Toshio Kuratomi <tkuratom@redhat.com>
+# License: GPLv3+
+# Copyright: Ansible Project, 2020
+"""General functions for working with aiohttp."""
```

Comment:
Docstring copy and paste error ;-)

Reply:
Fixed in 19c865f8344dd1cddf8caeefd46206496543a173.